### PR TITLE
Prevent error when the Vue instance was not initialized with i18n

### PR DESCRIPTION
**Pull request by mtorromeo**
This worked up to version 0.6.2
I have an application with 2 distinct Vue instances and only one of them requires the use of i18next and I am getting an error in the second instance.

Thanks

### CODE REVIEW

**Comment by claudiocro** [comment]
Thank you very much!

In this project we use semantic-release. Can you rephrase your commit message to `fix: prevent error when the Vue instance was not initialized with i18n??

FYI: [commit message format](https://github.com/semantic-release/semantic-release#default-commit-message-format)

Thx!

**Comment by mtorromeo** [comment]
@claudiocro Commit message updated
